src/lib/env.js

```diff
@@ -6,68 +6,56 @@
  */
 'use strict';
 
-let settings = {
-	os_auth_url: process.env.HUBOT_OBJECT_STORAGE_AUTH_URL,
-	os_user_id: process.env.HUBOT_OBJECT_STORAGE_USER_ID,
-	os_password: process.env.HUBOT_OBJECT_STORAGE_PASSWORD,
-	os_project_id: process.env.HUBOT_OBJECT_STORAGE_PROJECT_ID,
-	os_bluemix_region: process.env.HUBOT_OBJECT_STORAGE_BLUEMIX_REGION || 'dallas',
-	nlc_url: process.env.VCAP_SERVICES_NATURAL_LANGUAGE_CLASSIFIER_0_CREDENTIALS_URL || process.env.HUBOT_WATSON_NLC_URL,
-	nlc_username: process.env.VCAP_SERVICES_NATURAL_LANGUAGE_CLASSIFIER_0_CREDENTIALS_USERNAME || process.env.HUBOT_WATSON_NLC_USERNAME,
-	nlc_password: process.env.VCAP_SERVICES_NATURAL_LANGUAGE_CLASSIFIER_0_CREDENTIALS_PASSWORD || process.env.HUBOT_WATSON_NLC_PASSWORD,
-	nlc_objectstorage_classifier: process.env.HUBOT_OBJECT_STORAGE_SEARCH_CLASSIFIER_NAME ||
-		'cloudbot-obj-storage-classifier',
-	nlc_search_confidence_min: parseFloat(process.env.HUBOT_OBJECT_STORAGE_SEARCH_CONFIDENCE_MIN) || 0.0,
-	nlc_search_result_limit: parseInt(process.env.HUBOT_OBJECT_STORAGE_SEARCH_RESULT_LIMIT, 10) || 3,
-	nlc_classifier_cleanup_interval: parseInt(process.env.HUBOT_OBJECT_STORAGE_CLASSIFIER_CLEANUP_INTERVAL, 10) || 1000 *
-		60 * 60,
-	max_file_size: (1024 * 1024),
-	supported_adapters: ['slack', 'shell'],
-	nlc_version: 'v1'
-};
-
-// nlc service bound to application, overrides any other settings.
-if (process.env.VCAP_SERVICES && JSON.parse(process.env.VCAP_SERVICES).natural_language_classifier) {
```

Comment:
We need to follow this pattern in env.js of crawler project.  This was a recent change done by @ramaboggarapu I've created an issue in our crawler project:

https://github.com/ibm-cloud-solutions/hubot-ibmcloud-objectstorage-crawler/issues/16